Sources/SwerverCore/Router/Route.swift

```diff
@@ -15,7 +15,7 @@ public class Route {
         for (method, _) in actions {
             listOfMethods.append(method.rawValue)
         }
-        return listOfMethods
+        return listOfMethods.sorted(by: <)
```

Comment:
It looks like you are sorting both here and in the options action. Are both `sorted` calls needed?